### ISSUE TITLE
Recording vaccines for a child in the out of catchment area does not update the child's record on the tablet. 

### ISSUE THREAD

**Issue by faith-mutua** [OpenSRP/opensrp-client-path#137]
When recording vaccines for a child registered on the tablet in the out of catchment form, the record for this child does not update on the tablet.

**Comment by cagulas** [comment]
For some context on this, this was reported in the KIP app. This isn't the correct workflow, i.e. users should search the kid's record who is already on the tablet and not fill in the out of area form, but since the out of area form captures the kid's ZEIR ID, theoretically-speaking, it should be able to update the record on the server and then sync back to the same tablet, right?

**Comment by alaboso** [comment]
I am not sure if I should comment on this but the issues we had in KIP are 2 but very similar.
1. In current facility, facility b, open the form for recording an out of catchment vaccination. Enter KIP_ID for a child registered in facility a, fill the other details, submit. From current facility, b, go to the other facility, a, and search for the child using the KIP_ID, vaccine offered above isn't recorded.
2. Using the advanced search form, search for an out of catchment child. From the list, select and record a vaccine. Go to the child's facility and search for the child. The child's vaccination records do not include the vaccination recorded above.

**Comment by keymane** [comment]
For option 2, we should disable selecting from list if the record is from out of catchment. We don't need to show the vaccine card for an out of area record since we do not sync vaccines when searching.

**Comment by cagulas** [comment]
I assumed "from the list" meant the vaccines from the list of vaccines in the out of catchment vaccination form.

**Comment by keymane** [comment]
I meant the result list returned after searching for an out of catchment child. We should disable displaying the vaccine card for an out of area child

**Comment by cagulas** [comment]
Yes, that makes sense to me. Should we create a separate issue for this to implement?
